package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "csharp",
-  "version": "1.17.0-beta6",
```

Comment:
I can undo this, but saw that it was a little messed up?

Reply:
Arguably  it should be 1.17.1?

Reply:
Though I don't think this file gets used in any releasea activities.